### ISSUE TITLE
language not change

### ISSUE THREAD

**Issue by chitranjank** [Abedalkareem/LanguageManager-iOS#30]
label text not change when I language changed the application and these issue I faced in iOS 13

**Comment by irfanpule** [comment]
yes, I have same problem

**Comment by Abedalkareem** [comment]
Hi,
Can you please answer the questions below?
-Is it just on iOS 13?
-Did you follow the read me file?
-Do you use ```SceneDelegate```?
-Did you set the defaultLanguage? and which language was it?

can you share the code you are changing the language with?

thank you!

**Comment by irfanpule** [comment]
-Is it just on iOS 13?
`I just try on iOS 13 and 12, don't work`
-Did you follow the read me file?
`Yes, I am follow`
-Do you use SceneDelegate?
`No`
-Did you set the defaultLanguage? and which language was it?
`Yes I set the default Language and try don't set, the result it's same (not work)`
-Are you reloading the view controller after changing the language?
`Yes I reload`
-Can you share a screenshot before and after changing the language, or a video?

```
var lang: Languages!
       if language == "Indonesia" {
            lang = .id
        } else {
            lang = .en
        }
let tabBarController = getTabBarController()
        tabBarController.selectedIndex = 0
        LanguageManager.shared.setLanguage(language: lang, rootViewController: tabBarController, animation: { view in
            view.transform = CGAffineTransform(scaleX: 2, y: 2)
            view.alpha = 0
        })
```

**Comment by irfanpule** [comment]
Sorry this my mistake, I dont know why my project don't work. I try clone your repo and run Example project on iOS 13 and then work. I try add new language, yah it's work.

**Comment by irfanpule** [comment]
Yeh, I am very happy it's work on my project. Any problem when reload viewController. it's solve fo me

**Comment by krutikvora** [comment]
i have same issue in xcode 11.2.1 (ios 13.2)
i used SceneDelegate in my project
Please help me

**Comment by Abedalkareem** [comment]
Hi,
This issue now fixed in version 1.2.0.

Thank you!
